media-gfx/qimgv/qimgv-0.8.2.ebuild

```diff
@@ -14,23 +14,25 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="kde video"
 
-RDEPEND="
+BDEPEND="
+	virtual/pkgconfig
+"
+DEPEND="
 	dev-qt/qtconcurrent:5
 	dev-qt/qtcore:5
 	dev-qt/qtgui:5
 	dev-qt/qtwidgets:5
 	kde? ( kde-frameworks/kwindowsystem:5 )
 	video? ( media-video/mpv[libmpv] )
 "
-DEPEND="
-	${RDEPEND}
-	virtual/pkgconfig
+RDEPEND="
```

Comment:
`RDEPEND="${DEPEND}"`, I see you haven't really understood what BDEPEND is, maybe read some documentation on it? ;)